src/plugins/intel_gpu/src/runtime/debug_configuration.cpp

```diff
@@ -121,6 +122,7 @@ static void print_help_messages() {
     message_list.emplace_back("OV_GPU_ForceImplType", "Force implementation type of a target primitive or layer. [primitive or layout_name]:[impl_type]"
                               "For primitives, fc:onednn, fc:ocl, do:cpu, do:ocl, reduce:ocl and reduce:onednn are supported");
     message_list.emplace_back("OV_GPU_MaxKernelsPerBatch", "Maximum number of kernels in a batch during compiling kernels");
+    message_list.emplace_back("OV_GPU_DeviceID", "Set device id");
```

Comment:
I think I already asked this question, but let me do that again: Why do we need debug option for device ID? This it NOT debug capability, this is just feature required for our tests infra to allow running on non-default device. So I'd say that we need to add command line args for test apps. In unit tests it is added in this OR, so let's just do it for functional tests and that would be enough.

Reply:
I already answered in other thread, but let me paste it here: This is for the situation where we cannot specify target device id such as accuracy checker... The correct way would be to change accuracy checker but this will be easier path.